Makefile

```diff
@@ -61,26 +61,37 @@ bin/brogue.exe: $(objects) windows/icon.o
 	mt -manifest windows/brogue.exe.manifest '-outputresource:bin/brogue.exe;1'
 
 clean:
-	$(RM) src/brogue/*.o src/platform/*.o bin/brogue{,.exe}
+	$(RM) src/brogue/*.o src/platform/*.o windows/icon.o bin/brogue{,.exe}
 
 
-common-files := README.txt CHANGELOG.txt LICENSE.txt seed-catalog.txt
-common-bin := bin/assets bin/keymap.txt
+# Release archives
 
-%.txt: %.md
-	cp $< $@
+common_bin := bin/assets bin/keymap.txt
 
-windows.zip: $(common-files) $(common-bin)
-	zip -rvl $@ $^ bin/brogue.exe bin/*.dll bin/brogue-cmd.bat
+define make_release_base
```

Comment:
I notice the seed catalog is still not making it in..

Reply:
I think I'm gonna upload that as a separate download. It's easy to recreate or customise now

Reply:
I'll defer to your judgement, my main interest is in having a clear way for a dev to hand a test build to a tester that
* has the relevant git commit sha in it somewhere.
* is automated
* is in a format they would recognize

Thankfully the first point is already done and this PR should lock down the 2nd.   Using the same format as the release archives is an obvious way to get the third, but not the only way.

I do place a lot of value, however, on making it whatever solution we choose for point 3 to be just as dirt-easy as possible.  Double-clicking a zip and double-clicking the game inside is about right.  Having to e.g. move or rename binaries or whatever is already too much for some folk.